src/crypto/index.ts

```diff
@@ -610,6 +610,15 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         this.backupManager.checkAndStart();
     }
 
+    /**
+     * Implementation of {@link CryptoApi#getVersion}.
+     */
+    public getVersion(): null {
+        // `Crypto.getOlmVersion()` is used for the old crypto
```

Comment:
The old crypto version is set in this way https://github.com/matrix-org/matrix-js-sdk/blob/develop/src/client.ts#L2281 and we are using it here https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/components/views/settings/tabs/user/HelpUserSettingsTab.tsx#L75

I keep for the old crypto, the `olmVersion` field and I'm adding a new way to get the version for the new crypto (used here https://github.com/matrix-org/matrix-react-sdk/pull/11785/files#diff-c612bdd14b2fc22382b3675abd80d0b85f887453389d2a4c5b0fdcf2df887bd2R88).

Reply:
could you make this return something like `Olm ${olmVersionTuple[0]}.${olmVersionTuple[1]}.${olmVersionTuple[2]}`, so that both `Crypto` implementations have a `.getVersion` that returns a string, and just use that in the rust SDK?

Reply:
Yes good idea!